services/QuillLMS/app/models/concerns/vitally_shared_functions.rb

```diff
@@ -6,7 +6,9 @@ module VitallySharedFunctions
   attr_reader :vitally_entity, :school_year_start, :school_year_end
 
   PRE_DIAGNOSTIC_IDS = Activity.where(activity_classification_id: ActivityClassification.diagnostic&.id).where.not(follow_up_activity: nil).pluck(:id)
+  LIVE_PRE_DIAGNOSTIC_IDS = Activity.where(id: Activity::PRE_TEST_DIAGNOSTIC_IDS).pluck(:id)
```

Comment:
This isn't a huge deal, but doesn't this literally just go to the database to re-produce `Activity::PRE_TEST_DIAGNOSTIC_IDS`?  Is there a need to do a query and `pluck(:id)`?  Can we just use the constant here?